lib/screen/customer_support/support_thread_page.dart

```diff
@@ -662,3 +837,67 @@ class _SupportThreadPageState extends State<SupportThreadPage> {
     );
   }
 }
+
+class MyRatingBar extends StatefulWidget {
+  MyRatingBar({Key? key, required this.submit}) : super(key: key);
+  Future<dynamic> Function(String messageType, DraftCustomerSupportData data,
+      {bool isRating}) submit;
+
+  @override
+  _MyRatingBar createState() => _MyRatingBar();
+}
+
+class _MyRatingBar extends State<MyRatingBar> {
+  String customerRating = "";
+  int ratingInt = 0;
+  Widget sendButtonRating = SvgPicture.asset("assets/images/sendMessage.svg");
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.fromLTRB(0, 10, 0, 30),
+      color: Colors.black,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: [
+          RatingBar.builder(
+            minRating: 1,
+            itemSize: 24,
+            itemPadding: const EdgeInsets.symmetric(horizontal: 10.0),
+            itemBuilder: (context, _) => const Icon(
+              Icons.star,
+              color: Colors.white,
+            ),
+            unratedColor: const Color(0xff6D6B6B),
```

Comment:
consider this color also (if it existed in the theme) in other places (please recheck all)